tests/profiling/collector/test_stack.py

```diff
@@ -417,7 +418,7 @@ def test_exception_collection():
     assert e.sampling_period > 0
     assert e.thread_id == nogevent.thread_get_ident()
     assert e.thread_name == "MainThread"
-    assert e.frames == [(__file__, 411, "test_exception_collection", "")]
+    assert e.frames == [(__file__, 412, "test_exception_collection", "")]
```

Comment:
I added a new line to this file so I had to increment this line number. We should refactor this test but that's beyond the scope of this PR